src/lib.rs

```diff
@@ -432,6 +454,9 @@ fn mark_symbols(tree: &Tree, cursor: &mut QueryCursor, tla_lines: &mut [TlaLine]
 }
 
 fn replace_symbols(tla_lines: &mut [TlaLine]) {
+    if tla_lines.is_empty() {
+        return;
+    }
     for line_number in 0..tla_lines.len() - 1 {
```

Comment:
Instead of special-casing an empty input array, change this to:
```suggestion
    for line_number in 0..tla_lines.len().saturating_add_signed(-1) {
```